sim/core/stats/stats.go

```diff
@@ -311,16 +311,17 @@ type PseudoStats struct {
 	//  This includes almost all "(Normalized) Weapon Damage", but also some "School Damage (Physical)" abilities.
 	BonusDamage float64 // Comes from '+X Weapon Damage' effects
 
-	BonusRangedHitRating  float64 // Hit rating for ranged only.
-	BonusMeleeCritRating  float64 // Crit rating for melee only (not ranged).
-	BonusRangedCritRating float64 // Crit rating for ranged only.
-	BonusFireCritRating   float64 // Crit rating for fire spells only.
-	BonusShadowCritRating float64 // Crit rating for shadow spells only. Warlock stuff. You wouldn't understand.
-	BonusMHCritRating     float64 // Talents, e.g. Rogue Dagger specialization
-	BonusOHCritRating     float64 // Talents, e.g. Rogue Dagger specialization
-	BonusMHArmorPenRating float64 // Talents, e.g. Rogue Mace specialization
-	BonusOHArmorPenRating float64 // Talents, e.g. Rogue Mace specialization
-	BonusSpellCritRating  float64 // Crit rating bonus to spells
+	BonusRangedHitRating      float64 // Hit rating for ranged only.
+	BonusMeleeCritRating      float64 // Crit rating for melee only (not ranged).
+	BonusRangedCritRating     float64 // Crit rating for ranged only.
+	BonusFireCritRating       float64 // Crit rating for fire spells only.
+	BonusShadowCritRating     float64 // Crit rating for shadow spells only. Warlock stuff. You wouldn't understand.
+	BonusMHCritRating         float64 // Talents, e.g. Rogue Dagger specialization
+	BonusOHCritRating         float64 // Talents, e.g. Rogue Dagger specialization
+	BonusMeleeSpellCritRating float64 // Crit rating for melee special attacks, used for Warrior Recklessness
```

Comment:
If this stat is used only for warrior then it might not be a good idea to be a PseudoStat. 

Another way to implement this is to add a custom wrapper to the OutcomeApplier and modify the crit rating in there depending if reck is active or not